### ISSUE TITLE
I'll create a fantastic car parking system project using Arduino. 

### ISSUE THREAD

**Issue by AdityaTheDev** [prathimacode-hub/IoT-Spot#76]
I'm a Gssoc22 contributor

**Comment by prathimacode-hub** [comment]
Yup, sure. But kindly use the issue template provided and include the issue details along with tool you'll use and approach you would follow to get it assigned. @AdityaTheDev

**Comment by SiddhantPawar03** [comment]
Hey, is anyone working on same issue?

**Comment by prathimacode-hub** [comment]
Nope @SiddhantPawar03. If you would like to work on this issue. You can create a new issue with this title.
